src/cascadia/ut_app/JsonUtilsTests.cpp

```diff
@@ -101,6 +101,38 @@ JSON_FLAG_MAPPER(JsonTestFlags)
     };
 };
 
+struct hstring_like
+{
+    std::string value;
+};

```

Comment:
What? Why?

Reply:
I mean, was there something in the test that made this unclear?